tests/infra/utils.py

```diff
@@ -1,31 +1,30 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the Apache 2.0 License.
 import infra.path
-import hashlib
-import os
-import subprocess
+from hashlib import sha256
+import infra.snp as snp
 
 
-def get_code_id(
-    enclave_type, enclave_platform, oe_binary_dir, package, library_dir="."
-):
+def get_measurement(enclave_type, enclave_platform, package, library_dir="."):
     lib_path = infra.path.build_lib_path(
         package, enclave_type, enclave_platform, library_dir
     )
 
-    if enclave_platform == "sgx":
-        res = subprocess.run(
-            [os.path.join(oe_binary_dir, "oesign"), "dump", "-e", lib_path],
-            capture_output=True,
-            check=True,
-        )
-        lines = [
-            line
-            for line in res.stdout.decode().split(os.linesep)
-            if line.startswith("mrenclave=")
-        ]
+    if enclave_platform == "virtual":
+        hash = sha256(open(lib_path, "rb").read())
+        return hash.hexdigest()
 
-        return lines[0].split("=")[1]
     else:
-        # Virtual and SNP
-        return hashlib.sha256(lib_path.encode()).hexdigest()
+        raise ValueError(f"Cannot get measurement on {enclave_platform}")
+
+
+def get_host_data_and_security_policy(enclave_platform):
+    DEFAULT_VIRTUAL_SECURITY_POLICY = "Default CCF virtual security policy"
+    if enclave_platform == "snp":
+        security_policy = snp.get_container_group_security_policy()
+    elif enclave_platform == "virtual":
+        security_policy = DEFAULT_VIRTUAL_SECURITY_POLICY
+    else:
+        raise ValueError(f"Cannot get security policy on {enclave_platform}")
+    host_data = sha256(security_policy.encode()).hexdigest()
+    return host_data, security_policy
```

Comment:
Moving away from any ability to generate/check SGX attestations, updating terminology to "measurement" rather than "code_id". We get host data and security policies on the client for SNP (because we're actually the same box...), but don't yet get the measurement - I _think_ we could now do that from Python too, and get closer to an SNP code update story, but it's outside the scope of this PR.